template-manager-blueprint/src/main/java/com/sequenceiq/cloudbreak/blueprint/filesystem/FileSystemConfigQueryService.java

```diff
@@ -91,17 +92,18 @@ public void init() {
         return filtered;
     }
 
-    private String generateConfigWithParameters(String sourceTemplate, FileSystemType fileSystemType, Map<String, String> templateObject) throws IOException {
+    private String generateConfigWithParameters(String sourceTemplate, FileSystemType fileSystemType, Map<String, Object> templateObject) throws IOException {
         String defaultPath = fileSystemType.getDefaultPath();
-        Template defaultPathtemplate = handlebars.compileInline(defaultPath, HandlebarTemplate.DEFAULT_PREFIX.key(), HandlebarTemplate.DEFAULT_POSTFIX.key());
-        templateObject.put("defaultPath", defaultPathtemplate.apply(templateObject));
+        Template defaultPathTemplate = handlebars.compileInline(defaultPath, HandlebarTemplate.DEFAULT_PREFIX.key(), HandlebarTemplate.DEFAULT_POSTFIX.key());
+        templateObject.put("defaultPath", defaultPathTemplate.apply(templateObject));
         Template template = handlebars.compileInline(sourceTemplate, HandlebarTemplate.DEFAULT_PREFIX.key(), HandlebarTemplate.DEFAULT_POSTFIX.key());
         return template.apply(templateObject);
     }
 
-    private Map<String, String> getTemplateObject(FileSystemConfigQueryObject fileSystemConfigQueryObject, String protocol) {
-        Map<String, String> templateObject = new HashMap<>();
+    private Map<String, Object> getTemplateObject(FileSystemConfigQueryObject fileSystemConfigQueryObject, String protocol) {
+        Map<String, Object> templateObject = new HashMap<>();
         templateObject.put("clusterName", fileSystemConfigQueryObject.getClusterName());
+        templateObject.put("attachedCluster", fileSystemConfigQueryObject.isAttachedCluster());
```

Comment:
@sidseth the value added here is a `Boolean`. I tried using a `String` representation, but the handlebars helpers `if-true` / `if-false` require booleans, otherwise a `ClassCastException` will be thrown. Hence the change in the `Map` value type to `Object`. (I do not personally like this approach as it is inherently type-unsafe, but found no better way to resolve this.)